umm-spec-lib/resources/json-schemas/variable/umm/v1.8.1/umm-var-search-results-json-schema.json

```diff
@@ -67,34 +67,181 @@
+    "AssociationsInItemType": {
       "type": "object",
       "additionalProperties": false,
-      "description": "Concepts associated with the variable.",
+      "description": "Any concept associated with the service.",
       "properties": {
+        "variables": {
+          "description": "The concept ids of variables assciated with the service",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConceptRevisionType"
+          },
+          "minItems": 1
+        },
         "collections": {
-          "description": "The collections associated with the variable.",
+          "description": "The concept ids of collections assciated with the service",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConceptRevisionType"
+          },
+          "minItems": 1
+        },
+        "tools": {
+          "description": "The concept ids of tools assciated with the service",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConceptRevisionType"
+          },
+          "minItems": 1
+        },
+        "dataqualitysummaries": {
+          "description": "The concept ids of data quality summary concepts assciated with the service",
           "type": "array",
           "items": {
             "$ref": "#/definitions/ConceptRevisionType"
           },
           "minItems": 1
         },
-        "generics": {
-          "description": "The generic concepts associated with the variable.",
+        "orderoptions": {
+          "description": "The concept ids of order option concepts assciated with the service",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConceptRevisionType"
+          },
+          "minItems": 1
+        },
+        "serviceoptions": {
+          "description": "The concept ids of service option concepts assciated with the service",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConceptRevisionType"
+          },
+          "minItems": 1
+        },
+        "serviceentries": {
+          "description": "The concept ids of service entry concepts assciated with the service",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConceptRevisionType"
+          },
+          "minItems": 1
+        },
+        "grids": {
+          "description": "The concept ids of grid concepts assciated with the service",
           "type": "array",
           "items": {
             "$ref": "#/definitions/ConceptRevisionType"
           },
           "minItems": 1
         }
-      },
-      "required": ["collections"]
+      }
+    },
+    "AssociationsType": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "Any concept associated with the service.",
+      "properties": {
+        "variables": {
+          "description": "The concept ids of variables assciated with the service",
```

Comment:
typo, you can probably just do find and replace on string "assciated" for all of these.